### ISSUE TITLE
Fix `uncheckedSkip` document 

### ISSUE THREAD

**Issue by winterland1989** [GaloisInc/cereal#56]
Should `uncheckedSkip`'s document be `Skip ahead up to n bytes...`? because it seems it will not skip enough bytes if current chunk's length is not enough. I also suggest to make `unchecked..` version's document be more explicit about this behavior:

```
-- | Skip ahead up to @n@ bytes until end of this chunk. No error if there isn't enough bytes.
uncheckedSkip :: Int -> Get ()
uncheckedSkip n = do
    s <- get
put (B.drop n s)

-- | Get the next up to @n@ bytes as a ByteString until end of this chunk, without consuming them.
uncheckedLookAhead :: Int -> Get B.ByteString
uncheckedLookAhead n = do
    s <- get
return (B.take n s)
``` 

Does this make sense to you?

**Comment by elliottt** [comment]
I think this is a completely reasonable suggestion :)

I'll update the docs tomorrow. Thanks!

**Comment by elliottt** [comment]
This is fixed in d7c1ee0
